apps/docs/src/app/platform/component-docs/platform-action-bar/platform-action-bar-examples/platform-action-bar-with-back-button-example.component.html

```diff
@@ -0,0 +1,8 @@
+<fdp-action-bar
+    [title]="'Page Action bar component'"
+    (backButtonClick)="onBackBuutonClick()"
```

Comment:
Typo: "onBackBuutonClick" should be "onBackButtonClick"

Reply:
resolved